packages/app-desktop/gui/ConfigScreen/controls/plugins/PluginBox.tsx

```diff
@@ -137,9 +137,10 @@ export default function(props: Props) {
 	const item = props.item ? props.item : manifestToItem(props.manifest);
 
 	const onNameClick = useCallback(() => {
-		if (!props.item.manifest.homepage_url) return;
-		bridge().openExternal(props.item.manifest.homepage_url);
-	}, [props.item]);
+		const manifest = props.item ? props.item.manifest : props.manifest;
+		if (!manifest.homepage_url) return;
+		bridge().openExternal(manifest.homepage_url);
+	}, [props.item, props.manifest]);
```

Comment:
Instead of "props.item", could you directly use the "item" variable defined above?

Reply:
Done